<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Suggest::Widget</class>
 <widget class="QWidget" name="Suggest::Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>1537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="Suggest::TreeView" name="argumentSuggestTree"/>
   </item>
   <item row="1" column="1">
    <widget class="QTreeView" name="argumentPatchTree"/>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="typedMessageIcon">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="typedMessageInfo">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;TYPED MESSAGES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="typedMessageTransferButton">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="Suggest::TreeView" name="typedMessageSuggestTree"/>
   </item>
   <item row="3" column="1">
    <widget class="QTreeView" name="typedMessagePatchTree"/>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="attributeIcon">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nameMessageIcon">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="namedMessageInfo">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;ATTRIBUTES &amp;amp; NAMED MESSAGES &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="namedMessageTransferButton">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="Suggest::TreeView" name="namedMessageSuggestTree"/>
   </item>
   <item row="5" column="1">
    <widget class="QTreeView" name="namedMessagePatchTree"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="argumentIcon">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="argumentInfo">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;ARGUMENTS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="argumnetTransferButton">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Suggest::TreeView</class>
   <extends>QTreeView</extends>
   <header>SuggestTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
